module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -104,6 +104,19 @@ public function translate($target, $tokens = [], $default = null)
         // Figure out the text domain for the string:
         [$domain, $str] = $this->extractTextDomain($target);
 
+        if ($this->getTranslatorLocale() == "debug") {
+            $targetString = $domain !== "default" ? "$domain::$str" : "$str";
```

Comment:
Very minor nitpick, but you don't need double quotes around `"$str"` -- `$str` will do the same thing more efficiently. :-)

Reply:
...and if you want to be _really_ picky, it might also be slightly more efficient to use single quotes instead of double quotes for simple strings like `"debug"` that don't require interpolation. I _try_ to make a habit of this, though I don't claim that the code is currently 100% consistent. It would be nice to find a php-cs-fixer rule to do this for us someday! ;-)

Reply:
Cleaned this up. Although its debatable if single quotes are really more efficient its definitely nice to have this consistent. I will keep this in mind in the future.

Reply:
Thanks, @ThoWagen. And thanks to #3035, this will soon be automated for us!